scripts/userstyles.yml

```diff
@@ -655,6 +657,14 @@ userstyles:
     readme:
       app-link: "https://www.npmjs.com/"
     current-maintainers: [*uncenter]
+  ollama:
+    name: Ollama
+    categories: [development]
+    icon: ollama
```

Comment:
Doesn't exist yet, remove for now and I'll add it back later when the icon exists.
```suggestion
```